test/classes/delete-free/lifetimes/field-init-escapes.skipif

```diff
@@ -0,0 +1 @@
+CHPL_TEST_VGRND_EXE != on
```

Comment:
I usually just make such tests .noexec and make the .bad file be empty (as in, it shouldn't have compiled).

Reply:
Oh you're right, that sounds a lot better!  I don't think I've used that pattern (definitely not recently anyway).